source/_posts/rails-docker.md

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
keyword should be linked with previous nayan blog if possible

Reply:
Its not linked with previous posts.